test/ClientTest.php

```diff
@@ -11,25 +11,27 @@
 
 class ClientTest extends HttpTestCase
 {
-
 	public function testGetRequest()
 	{
 		$client = new Net_Http_Client();
 		$client->get(self::HOST.'/basic/get');
 		$this->assertEquals(200, $client->getStatus());
-		$this->assertContains("CANHAZHTTPGET", $client->getBody());
+		$this->assertStringContainsString("CANHAZHTTPGET", $client->getBody());
 	}
 
 	public function testPostRequest()
 	{
 		$client = new Net_Http_Client();
 		$client->post(self::HOST.'/basic/post', array("greeting"=>"Hello", "from"=>"Net_Http_Client"));
 		$this->assertEquals(200, $client->getStatus());
-		$this->assertContains("Hello back", $client->getBody());
+		$this->assertStringContainsString("Hello back", $client->getBody());
 	}
 
 	public function testHeadRequestWithSetHeader()
 	{
+		// The host used for these tests no longer returns the expected headers
+		$this->markTestSkipped();
```

Comment:
Given this library hasn't been touched in 9 years, I wonder if there's any value keeping these tests if they don't work...

Reply:
I've deleted them since it's unlikely that we'll fix the host